src/room/track/RemoteVideoTrack.ts

```diff
@@ -144,6 +144,11 @@ export default class RemoteVideoTrack extends RemoteTrack {
     return detachedElements;
   }
 
+  /** @internal */
+  getDecoderImplementation(): string {
```

Comment:
nit: avoid returning empty string when we don't know the answer

```suggestion
  getDecoderImplementation(): string | undefined {
```